raster-foundry/raster-foundry
# Add single band options to project layers

## Overview

This PR adds single band info to project layers.

### Checklist

- [x] [Swagger specification](https://github.com/raster-foundry/raster-foundry-api-spec) updated (raster-foundry/raster-foundry-api-spec#93)

## Testing Instructions

 * set a project to single band viz
 * run migration
 * should still be able to view it as single band visualized
 * change the SBOs (color mode or band, whatever)
 * the visualization should reflect your changes

Closes #4510 

Closes #4715 (by accident? Something horrible was happening with decoders and the circe upgrade seems to have fixed #4715 as well)
